apps/api/v2/src/ee/calendars/services/gcal.service.ts

```diff
@@ -10,7 +10,8 @@ import { Injectable } from "@nestjs/common";
 import { ConfigService } from "@nestjs/config";
 import { Prisma } from "@prisma/client";
 import { Request } from "express";
-import { google } from "googleapis";
+import { OAuth2Client } from "google-auth-library";
+import { calendar_v3 } from "googleapis";
```

Comment:
Using @googleapis/admin, calendar, we also optimise packaging 💪

Reply:
Done. Seems like we can import from `@googleapis/admin` but not from `@googleapis/calendar`.

Reply:
We can 💪 (got that working locally, but did have to yarn add)

Reply:
Oh really? Nice, maybe it's just a type error. Do you get a type error as well?

Reply:
<img width="420" alt="Screenshot 2024-11-23 at 3 34 39 AM" src="https://github.com/user-attachments/assets/7043589b-6341-423f-af7d-4279df4646a7">

This is what my `@googleapis` looks like in node_modules

Reply:
Yeah they're separate installs, we can actually remove googleapis after

Reply:
ok, good call